community/kernel-api/src/test/java/org/neo4j/internal/kernel/api/NodeValueIndexCursorTestBase.java

```diff
@@ -280,15 +311,157 @@ public void shouldPerformIndexScan() throws Exception
         // given
         int label = token.nodeLabel( "Node" );
         int prop = token.propertyKey( "prop" );
-        IndexReference index = schemaRead.index( label, prop );
+        CapableIndexReference index = schemaRead.index( label, prop );
+        IndexValueCapability wildcardCapability = index.value( new ValueGroup[]{null} );
```

Comment:
The reason we want to use an explicit null value in array is because of composite indexes where you can have something like `new ValueGroup[]{null,ValueGroup.TEXT}`, indicating that first part of key could be of any value group and the second part is known to be a string.

Do you still think it is reasonable to use empty array for single property wildcard? I can see the use for that syntactic sugar but at the same time I don't like multiple ways of doing the same thing.

I guess another alternative would be to add another `ValueGroup.WILDCARD` or use `ValueGroup.NO_VALUE` for that purpose. But doesn't seem right to pollute `Values` module like that.

Reply:
We have a `ValueGroup.NO_VALUE` already. Maybe we should add `ValueGroup.ANY`?

Reply:
I'm gonna try that 🎉